packages/ra-ui-materialui/src/input/InputHelperText.tsx

```diff
@@ -18,7 +18,9 @@ const InputHelperText: FunctionComponent<Props> = ({
         <ValidationError error={error} />
     ) : helperText ? (
         <>{translate(helperText, { _: helperText })}</>
-    ) : null;
+    ) : (
```

Comment:
I don't understand that change. What's wrong with `null`?

Reply:
With null, the helper component is not rendered
With the empty fragment, it is rendered with nothing inside